doc/controls/TabBar.md

```diff
@@ -0,0 +1,184 @@
+
+### Properties
+Properties|Type|Description
+-|-|-
+SelectedItem|object|Gets or sets the selected `TabBarItem`.
+SelectedIndex|int|Gets or sets the index of the selected `TabBarItem`.
+
+> Note: `TabBar` only supports single selection at this time.
+
+### Events
+All events below are forwarded from the nested `TabBarItem`s:
+Events|Type|Description
+-|-|-
+SelectionChanged|`TypedEventHandler<TabBar, TabBarSelectionChangedEventArgs>`|Occurs when a `TabBarItem`'s selection state has changed.
+
+```cs
+class TabBarSelectionChangedEventArgs : EventArgs
+{
+    // Gets the newly selected TabBarItem
+    object? NewItem { get; }
+
+    // Gets the previously selected TabBarItem
+    object? OldItem { get; }
+}
+```
+
+## Styling `TabBar` & `TabBarItem`
+The default styles for `TabBar` and `TabBarItem` are very basic and it is assumed that they will have their styles set to either:
+* One of the pre-built styles that come packaged within the `Uno.Toolkit.UI.Material` or `Uno.Toolkit.UI.Cupertino` libraries
+* A custom built style that is defined within the consuming application
+
+The styles that exist within the Toolkit Theming Libraries are built for the most common scenarios in which a `TabBar` could be used.
+
+### "Top" `TabBar` Style
+
+A common use for a `TabBar` would be to use it as a means of navigation between two or more views/pages that are all at the same level of hierarchy (Lateral Navigation). If the goal is provide lateral navigation for destinations that are at **any** level of hierachy, a `TabBar` styled as a `TopTabBar` is most appropriate. 
+
+Currently, the Material Theme Toolkit Library contains two styles for this purpose: `ColoredTopTabBarStyle` and `TopTabBarStyle`.
+ 
+![](../assets/tabbar-android-top.png)
+![](../assets/tabbar-ios-top.png)
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:TabBar SelectedIndex="1"
+			Style="{StaticResource ColoredTopTabBarStyle}">
+	<utu:TabBar.Items>
+		<utu:TabBarItem Content="Home" />
+		<utu:TabBarItem Content="Search" />
+		<utu:TabBarItem Content="Support" />
+		<utu:TabBarItem Content="About" />
+	</utu:TabBar.Items>
+</utu:TabBar>
+```
+
+### "Bottom" `TabBar` Style
+
+Following the explanation of Lateral Navigation and the use case for "Top" `TabBar` styles, another common scenario is to use `TabBar` as an application's primary navigation component. In this case, we want a component that would provide lateral navigation to all **top-level** destinations. In this case, a `TabBar` styled as a `BottomTabBar` is most appropriate.
+
+Currently, both the Material and Cupertino Theme Toolkit Libraries contain a `BottomTabBarStyle`.
+
+#### Material
+
+![](../assets/tabbar-android-material-bottom.png)
+![](../assets/tabbar-ios-material-bottom.png)
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:TabBar Style="{StaticResource BottomTabBarStyle}">
+	<utu:TabBar.Items>
+		<utu:TabBarItem Content="Home">
```

Comment:
for the TabBar example, maybe use simpler `TabBarItem` like one-liner with just `Content` as text to get the idea across: you can nest `TabBarItem`s
and then add a few more example separately for `TabBarItem`, to show case what else is possible: icon and all IconElement variants, flyout...
or minimally, keep the flyout one last